src/mlj_catboostclassifier.jl

```diff
@@ -81,20 +81,24 @@ function MMI.fit(mlj_model::CatBoostClassifier, verbosity::Int, data_pool)
     cache = (; mlj_model=deepcopy(mlj_model))
     report = (feature_importances=feature_importance(model),)
 
-    return (model, cache, report)
+    fitresult = (model, y_first)
+
+    return (fitresult, cache, report)
 end
 
 MMI.fitted_params(::CatBoostClassifier, model) = (model=model,)
 MMI.reports_feature_importances(::Type{<:CatBoostClassifier}) = true
 
-function MMI.predict(mlj_model::CatBoostClassifier, model, X_pool)
+function MMI.predict(mlj_model::CatBoostClassifier, fitresult, X_pool)
+    model, y_first = fitresult
+    classes = MMI.classes(y_first)
     py_preds = predict_proba(model, X_pool)
-    classes = pyconvert(Array, model.classes_.tolist())
-    preds = MMI.UnivariateFinite(classes, pyconvert(Array, py_preds); pool=missing)
+    preds = MMI.UnivariateFinite(classes, pyconvert(Array, py_preds))
     return preds
 end
```

Comment:
The other option is to do `pool=y_first`. I'm  wondering about whether `classes` as defined here will necessarily have the same order as `classes` defined previously. The important thing is for the ordering in the first two arguments on the `UnivariateFinite` constructor be consistent. Maybe the original definition of `classes` is safer here??

Reply:
Actually, I think you *must* use my suggestive alternative here because the new `classes` will not always have the same length as the old python `classes`. I've just found new fails in the integration tests that suggested this to me.

Reply:
So, to be clear, use `classes` as defined  before your recent commits, but change `pool=missing` to `pool=y_first`.

Reply:
Just updated it to use the old method for `classes` with `pool=y_first`

Reply:
Resolved.